angular/projects/spark-angular/src/lib/components/sprk-award/sprk-award.component.ts

```diff
@@ -59,138 +55,173 @@ import { Component, Input } from '@angular/core';
         toggleType="base"
         title="{{ disclaimerTitle }}"
-        analyticsString="{{ analyticsStringDisclaimer }}"
+        analyticsString="{{
+          disclaimerAnalyticsString || analyticsStringDisclaimer
+        }}"
       >
         <p class="sprk-b-TypeBodyFour">{{ disclaimerCopy }}</p>
       </sprk-toggle>
     </sprk-stack>
-  `
+  `,
 })
 export class SprkAwardComponent {
   /**
-   * The relative size of the viewport that
-   * the Award component should switch
-   * from a stacked layout to a side by side
-   * layout. You will need to experiment
-   * with your content to see which value
-   * is the best fit. Can be `tiny`,
-   * `small`, `medium`, `large` or `huge`.
+   * The relative size of the viewport that the Award component should switch
+   * from a stacked layout to a side by side layout. You will need to
+   * experiment with your content to see which value is the best fit. Can
+   * be `tiny`, `small`, `medium`, `large` or `huge`.
    */
   @Input()
-  splitAt: string;
+  splitAt: 'tiny' | 'small' | 'medium' | 'large' | 'huge';
   /**
-   * The `alt` text that will be applied
-   * to the first image.
+   * This is used as the amount of spacing between the elements in the Award.
+   * The value supplied can be `tiny`, `small`, `medium`, `large`, or `huge`.
+   */
+  @Input()
+  itemSpacing: 'tiny' | 'small' | 'medium' | 'large' | 'huge' = 'medium';
+  /**
+   * The `alt` text that will be applied to the first image.
    */
   @Input()
   imgOneAlt: string;
   /**
-   * The image `href` value that will be
-   * applied to the first image.
+   * The image `href` value that will be applied to the first image.
    */
   @Input()
   imgOneHref: string;
   /**
-   * The image `href` value that will be
-   * applied to the second image.
+   * The image `href` value that will be applied to the second image.
    */
   @Input()
   imgTwoHref: string;
   /**
-   * The `alt` text that will be applied
-   * 'to the second image.
+   * The `alt` text that will be applied to the second image.
    */
   @Input()
   imgTwoAlt: string;
   /**
-   * The image source that will be
-   * applied to the first image.
+   * The image source that will be applied to the first image.
    */
   @Input()
   imgOneSrc: string;
   /**
-   * The image source that will be
-   * applied to the second image.
+   * The image source that will be applied to the second image.
    */
   @Input()
   imgTwoSrc: string;
+  // TODO - Remove as part of Issue 1279
   /**
-   * The string that will be assigned to the
-   * `data-analytics` attribute of the first image.
+   * Deprecated - use ` imgOneAnalyticsString` instead. The string that
+   * will be assigned to the `data-analytics` attribute of the first image.
    */
   @Input()
   analyticsStringImgOne: string;
   /**
-   * The string that will be assigned to the
-   * `data-analytics` attribute of the second image.
+   * The string that will be assigned to the `data-analytics` attribute of
```

Comment:
See https://github.com/sparkdesignsystem/spark-design-system/pull/3565#discussion_r508579033 for the changes needed for this description.